### PR TITLE
Invalid enum validation update

### PR DESCRIPTION
**Pull request by viralpandya**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -474,3 +474,74 @@ private void ValidateStudentId(Student student) { ... }
...
+## **Invalid Enum validations**
+
+Let's assume our Student model looks as follows:
+
+```cs
+public class Student {
+	StudentType Type {get; set;}
+}
+
+public enum StudentType{
```
Comment (hassanhabib) [inline README.md]: @viralpandya push the open bracket to the next line 
do the same for the above

File: README.md [hunk 2]
```diff
@@ -474,3 +474,74 @@ private void ValidateStudentId(Student student) { ... }
...
+
+## **Invalid Enum validations test*
+ ```csharp
+[Fact]
+public async Task ShouldThrowValidationExceptionOnRegisterIfStudentTypeIsInvalidAndLogItAsync()
+{
+	// given
+	Student randomStudent = CreateRandomStudent();
+	Student inputStudent = randomStudent;
+    StudentType invalidStudentType = GetInvalidStudentType();
```
Comment (hassanhabib) [inline README.md]: Fix indentation please

File: README.md (hunk 1 above)
Comment (viralpandya) [reply README.md]: Resolved

File: README.md (hunk 2 above)
Comment (viralpandya) [reply README.md]: Resolved

**Comment by hassanhabib** [comment]
image
@viralpandya  the visualization doesn't seem to be looking right.

**Comment by viralpandya** [comment]
Resolved

**Pull request merged**